client/blocks/signup-form/index.jsx

```diff
@@ -852,14 +860,19 @@ class SignupForm extends Component {
 			? this.getLoginLink()
 			: localizeUrl( config( 'login_url' ), this.props.locale );
 
+		const loginTextByFlowName =
+			flowName === 'onboarding'
+				? translate( 'Log in to create a site for your existing account.' )
+				: translate( 'Already have a WordPress.com account?' );
+
+		const footerLoginText = this.props.footerLoginText || loginTextByFlowName;
```

Comment:
As per this [comment](https://github.com/Automattic/wp-calypso/pull/43061#issuecomment-644730362), `footerLoginText` needs to be `Already have an account? Log in`.

Reply:
Fixed copy in the [steps-pure.js file](https://github.com/Automattic/wp-calypso/pull/43061/files#diff-ae0bf020974ae29c034739f287afc5c5R186)